schema/deploy/shipping/views.sql

```diff
@@ -2287,4 +2288,28 @@ grant select
     on shipping.seattle_neighborhood_districts_v1
     to "scan-dashboard-exporter";
 
+
+create or replace view shipping.scan_enrollments_v1 as
+
+    select
+        illness_questionnaire_date,
+        scan_study_arm,
+        priority_code,
+        puma,
+        neighborhood_district
+
+    from shipping.scan_encounters_v1
+;
```

Comment:
Because this view is only a subset of `shipping.scan_encounters_v1`, is it necessary? Can we just tell Formative which columns are relevant to them?

Reply:
I don't think we want Formative to have access to the entire `shipping.scan_encounters_v1` view.  I think it's easier to do that as a new view rather than maintaining grants to only specific columns of the view.

Reply:
Ah, ok. Yes this probably is easier then.